types/ember__helper/ember__helper-tests.ts

```diff
@@ -0,0 +1,7 @@
+import { array, concat, fn, get, hash } from '@ember/helper';
+
+array; // $ExpectType ArrayHelper
+concat; // $ExpectType ConcatHelper
+fn; // $ExpectType FnHelper
+get; // $ExpectType GetHelper
+hash; // $ExpectType HashHelper
```

Comment:
Given the only thing you can do with these values is reference them in a template, the most I think we can do in testing is assert that they're importable and have the expected type label.